Description: fix build with gcc 4.7
 Include necessary header for _exit()
Author: Ilya Barygin <randomaction@ubuntu.com>

--- a/app/clasp_app.cpp
+++ b/app/clasp_app.cpp
@@ -22,6 +22,7 @@
 #include <iostream>
 #include <fstream>
 #include <clasp/clause.h>
+#include <unistd.h>
 /////////////////////////////////////////////////////////////////////////////////////////
 // Application
 /////////////////////////////////////////////////////////////////////////////////////////
